infrastructure/modules/pipeline_step_lambda/main.tf

```diff
@@ -1,6 +1,6 @@
 
 locals {
-  ecr_image_tag     = var.namespace
```

Comment:
What's this change for?

Reply:
I reverted this change in favour of manually tagging existing ECR images with the new pipeline date